test/spec/modules/33acrossAnalyticsAdapter_spec.js

```diff
@@ -309,6 +310,7 @@ describe('33acrossAnalyticsAdapter:', function () {
           events.emit(EVENTS.BID_WON, bidWon);
         }
 
+        sandbox.clock.tick(this.defaultTimeout);
         assert.calledOnceWithStringJsonEquivalent(navigator.sendBeacon, endpoint, createReportWithThreeBidWonEvents());
```

Comment:
could you please add another assertion that test the opposite, before the clock tick.
```
// Add an assertion that check sendBeacon hasn't been called.
sandbox.clock.tick(this.defaultTimeout);
assert.calledOnceWithStringJsonEquivalent(navigator.sendBeacon, endpoint, createReportWithThreeBidWonEvents());
```

Reply:
Great catch. This was a faulty premise after this refactor. I have fixed the test.